edx_credentials_themes/static/mitpe/sass/_components-rendering.scss

```diff
@@ -23,7 +23,7 @@
     margin-bottom: 0;
 
     @include susy-breakpoint($bp-screen-md, $susy) {
-      font-size: rem(32);
+      font-size: 2rem;
```

Comment:
Should this also be a variable since we reuse it?

Reply:
The font-size scale we're using is getting carried over from edx-pattern library and it didn't make sense to define more size variables for every font size we used outside of the scale even if it was reused. Mike Terry and I discussed credentials mess of font-size sometimes being variables and sometimes not being variables and our conclusion was that these changes should migrate us out of pattern library and if we want to move away from hard coded rem values in the future we should fix that later.